omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb

```diff
@@ -1,70 +1,59 @@
 #
-# Author:: Adam Jacob (<adam@opscode.com>)
-# Copyright:: Copyright (c) 2011 Opscode, Inc.
+# Author:: Adam Jacob <adam@chef.io>
+# Copyright:: Copyright (c) 2011-2015 Chef Software, Inc.
```

Comment:
For future reference, if we're updating the license headers we should make them Apache Licensed. Although we do have language all over [the source](https://github.com/chef/chef-server/tree/master/src/oc_bifrost#license) saying that everything is Apache 2.